### PR TITLE
herbstluftwm: update to 0.9.4

### PR DESCRIPTION
**Pull request by Barbaross93**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by tibequadorian** [comment]
The first build was broken so you can't have tested it. Have you tested it now?

**Comment by Barbaross93** [comment]
Yes, I had made a small mistake. I've since tested it again and it now works.

**Comment by Barbaross93** [comment]
I'd be happy to be the maintainer for this package. Do I just go ahead and add myself as the maintainer?

**Comment by tibequadorian** [comment]
Sure, go ahead!

File: srcpkgs/herbstluftwm/template [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # Template file for 'herbstluftwm'
 pkgname=herbstluftwm
-version=0.9.3
+version=0.9.4
 revision=1
 build_style=cmake
 hostmakedepends="pkg-config asciidoc"
 makedepends="libXrandr-devel libXinerama-devel libglib-devel libXft-devel
-	libXrender-devel freetype-devel"
+	libXrender-devel freetype-devel libXfixes-devel"
```
Comment (paper42) [inline srcpkgs/herbstluftwm/template:8]:
```suggestion
 libXrender-devel freetype-devel libXfixes-devel"
```
nitpick

Comment (tibequadorian) [reply srcpkgs/herbstluftwm/template]: replace it with a space @Barbaross93

Comment (tibequadorian) [reply srcpkgs/herbstluftwm/template]: replace it with a space @Barbaross93

Comment (Barbaross93) [reply srcpkgs/herbstluftwm/template]: Ahh, my bad. Didn't see the space there at first.

**Review by tibequadorian — APPROVED** [review]

**Pull request merged**